### PR TITLE
🔧 Fix to add missing access controls

### PR DESCRIPTION
**Pull request by YutoMizutani**
Support `public` and `open` prefix in extensions. 
The PR includes some tests with generated codes using GitHub Actions. It can be found the wrong (false negative) cases, but it finds no any missing (false positive; no generating) case.
screenshot 605

### CODE REVIEW

File: Sourcery/templates/AutoPreviewable.stencil [hunk 1]
```diff
@@ -2,7 +2,7 @@ import UIKit
 #if canImport(SwiftUI) && DEBUG
 import SwiftUI
 
-{% for type in types.types|class where type.implements.AutoPreviewable or type|annotated:"AutoPreviewable" %}
+{% for type in types.types|class where type.actualTypeName.AutoPreviewable or type|annotated:"AutoPreviewable" %}
```
Comment (YutoMizutani) [inline Sourcery/templates/AutoPreviewable.stencil:5]: `actualTypeName` property can find from the whole targets

**Comment by AkkeyLab** [comment]
Thank you!
I review until this weekend.

File: AutoPreviewableTests/AccessControlTests.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import XCTest
+@testable import AutoPreviewable
+
+// sourcery: xibLess
+public final class AccessControlPublicView: UIView, AutoPreviewable {}
+// sourcery: xibLess
+internal final class AccessControlInternalView: UIView, AutoPreviewable {}
+// sourcery: xibLess
+final class AccessControlImplicitlyInternalView: UIView, AutoPreviewable {}
```
Comment (AkkeyLab) [inline AutoPreviewableTests/AccessControlTests.swift:17]: nits:
I think you can add a pattern called `open`.

**Review by AkkeyLab — APPROVED** [review]
LGTM !!

**Pull request merged**